### PR TITLE
FAQ for review

### PR DESCRIPTION
**Pull request by jschlyter**
Frequently asked questions written by @gabywh with edits from @jschlyter.

### CODE REVIEW

**Review by gabywh — APPROVED** [review]

File: FAQ.md
Comment (martin-lindstrom) [inline FAQ.md:51]: No examples of UVCI:s?

Did I miss the discussion about which characters that should be included for Luhn mod N?

File: FAQ.md
Comment (martin-lindstrom) [inline FAQ.md:65]: Add a sentence: "If more than one entry is included the list should be sorted with the most recent entry first". Or something like this? There are discussions where some state that only one entry should be included and other want to include several. In the cases several entries are included we will increase the chances of good interoperability if the most important entry (the most recent) is placed first in the list. Some verifiers maybe only reads the first entry.

**Review by martin-lindstrom — APPROVED** [review]
Approved, but check my comments/suggestions.

File: FAQ.md
Comment (gabywh) [reply FAQ.md:51]: No, not missed. I'm waiting / blocked by this review so I can add the Luhn-Mod-N text + example (@jschlyter really wanted the text in the wiki for some reason so now I can't edit the wiki until this reviewed, approved and merged)

**Review by kruzikh — APPROVED** [review]

File: FAQ.md
Comment (gabywh) [reply FAQ.md:65]: > "If more than one entry is included the list should be sorted with the most recent entry first"

Or last entry is most recent, also do-able.
Nothing in regulations or guidelines. I guess it is easy enough to sort on date if needed - and which date to sort on? Valid from? Valid until? Most recently entered? I guess us ordering internally is an option but has some (if limited) usefulness. Is this a blocking issue or can we take it up post v1.0.0 ?

**Comment by jschlyter** [comment]
merged to wiki

File: FAQ.md
Comment (martin-lindstrom) [reply FAQ.md:65]: Let's leave it out for now ...

**Pull request closed without merging**
